be/src/storage/rowset/segment_v2/bitshuffle_page.h

```diff
@@ -387,13 +396,16 @@ class BitShufflePageDecoder final : public PageDecoder {
 
     Status _decode_to(uint8_t* to) {
         char* in = const_cast<char*>(&_data[BITSHUFFLE_PAGE_HEADER_SIZE]);
-        int64_t bytes = bitshuffle::decompress_lz4(in, to, _num_element_after_padding, _size_of_element, 0);
-        if (PREDICT_FALSE(bytes < 0)) {
-            // Ideally, this should not happen.
-            LOG(ERROR) << "bitshuffle decompress failed: " << bitshuffle_error_msg(bytes);
-            return Status::RuntimeError("Unshuffle Process failed");
+        if (!_options.is_decoded) {
```

Comment:
can use _options.use_page_cache to replace the _options.is_decoded.

Reply:
> 

seems you are right, fix in https://github.com/StarRocks/starrocks/pull/1597/commits/07feaa31b181895da4c8324dc6262490b93698f6